themes/digital.gov/src/scss/new/_card-news.scss

```
@@ -43,7 +41,7 @@
     order: -1;
 
     img {
-      @include u-border("1px", "gray-cool-10", "solid");
+      border: 1px color("gray-cool-10") "solid";
```

Quotes aren't necessary for border style. Also, we should use theme color tokens to make color changes more manageable and code more readable.

```suggestion
      border: 1px color("base-lighter") solid;
```

---

Good call, updated.